scripts/run-browsers-smoketests.js

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env node
+const shell = require("shelljs");
+
+let result = 0;
+
+console.log(`
+Test webextension-polyfill on real browsers
+===========================================`);
+
+// Enable headless mode (currently only used when running on Firefox
+// because Chrome doesn't currently support the extensions in headless mode)
+process.env.HEADLESS = 1;
+
+console.log(`
+Runing smoketests on Chrome`);
+process.env.TEST_BROWSER_TYPE = "chrome";
+result = shell.exec("npm run test-integration:chrome").code || result;
```

Comment:
How about early exit with `process.exit(-1)` (or with `process.exit(execResult.code)`) on the first `shell.exec` that fails?

to be honest, shelljs has a `shell.set` method which could allow us to solve this scenarios as it was solved in the original bash script:
- https://github.com/shelljs/shelljs#setoptions

I never used it, but it seems to be worth a try.

@ExE-Boss do you mind to give a try to `shell.set`?

(same comment on run-module-bundlers-smoketests.js)

Reply:
As&nbsp;I added to&nbsp;the&nbsp;documentation of&nbsp;the&nbsp;`result` variable:

> This is to make sure that even if the tests fail on Chrome, the tests still run on Firefox, so that it can be seen whether Firefox broke too or is unaffected.